python/pylibcudf/pylibcudf/io/json.pyx

```diff
@@ -323,21 +355,81 @@ cdef class JsonReaderOptionsBuilder:
         self.c_obj.compression(compression)
         return self
 
-    cpdef JsonReaderOptionsBuilder lines(self, bool val):
+    cpdef JsonReaderOptionsBuilder dayfirst(self, bool val):
         """
-        Set whether to read the file as a json object per line.
+        Set whether the reader should parse dates as DD/MM versus MM/DD.
 
         Parameters
         ----------
         val : bool
-            Boolean value to enable/disable the option
-            to read each line as a json object
+            Boolean value to indicate whether the
+            reader should enable/disable DD/MM parsing
 
         Returns
         -------
         Self
         """
-        self.c_obj.lines(val)
+        self.c_obj.dayfirst(val)
+        return self
+
+    cpdef JsonReaderOptionsBuilder delimiter(self, char delimiter):
```

Comment:
```suggestion
    cpdef JsonReaderOptionsBuilder delimiter(self, str delimiter):
```
We typically use `str` to annotate string inputs in pylibcudf instead of `char`